cargo-dist/src/config/v0.rs

```diff
@@ -467,6 +467,11 @@ pub struct DistMetadata {
     #[serde(skip_serializing_if = "Option::is_none")]
     #[serde(default)]
     pub cargo_cyclonedx: Option<bool>,
+
+    /// Whether to generate OmniBOR artifact IDs.
+    #[serde(skip_serializing_if = "Option::is_none")]
+    #[serde(default)]
+    pub omnibor: Option<bool>,
```

Comment:
I wonder if we want to name this something more specific? Like `generate-omnibor-id`.